docs/docs/install/environment-variables.md

```diff
@@ -60,14 +60,17 @@ These environment variables are used by the `docker-compose.yml` file and do **N
 
 ## Database
 
-| Variable      | Description       |   Default   | Services              |
-| :------------ | :---------------- | :---------: | :-------------------- |
-| `DB_URL`      | Database URL      |             | server, microservices |
-| `DB_HOSTNAME` | Database Host     | `localhost` | server, microservices |
-| `DB_PORT`     | Database Port     |   `5432`    | server, microservices |
-| `DB_USERNAME` | Database User     | `postgres`  | server, microservices |
-| `DB_PASSWORD` | Database Password | `postgres`  | server, microservices |
-| `DB_DATABASE` | Database Name     |  `immich`   | server, microservices |
+| Variable                         | Description                                                   |   Default    | Services              |
+| :------------------------------- | :------------------------------------------------------------ | :----------: | :-------------------- |
+| `DB_URL`                         | Database URL                                                  |              | server, microservices |
+| `DB_HOSTNAME`                    | Database Host                                                 | `localhost`  | server, microservices |
+| `DB_PORT`                        | Database Port                                                 |    `5432`    | server, microservices |
+| `DB_USERNAME`                    | Database User                                                 |  `postgres`  | server, microservices |
+| `DB_PASSWORD`                    | Database Password                                             |  `postgres`  | server, microservices |
+| `DB_DATABASE`                    | Database Name                                                 |   `immich`   | server, microservices |
+| `VECTOR_EXTENSION`<sup>\*1</sup> | Database Vector Extension (one of [`pgvector`, `pgvecto.rs`]) | `pgvecto.rs` | server, microservices |
```

Comment:
Also, all of the other database env variables are prefixed with `DB_`. Should this one be the same?